zsh-vim-mode.plugin.zsh

```diff
@@ -118,150 +118,151 @@ function vim-mode-bindkey () {
     done
 }
 
+if [[ -z $ZSH_VIM_MODE_NO_DEFAULT_BINDINGS ]]; then
```

Comment:
I've use `VIM_MODE_X`, without the `ZSH_` prefix, for settings. I'd prefer to use that here, too.